.github/workflows/build.yaml

```diff
@@ -333,16 +326,19 @@ jobs:
       GITHUB_TOKEN: ${{ secrets.GH_RAD_CI_BOT_PAT }}
     steps:
       - name: Checkout
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
       - name: Parse release version and set environment variables
         run: python ./.github/scripts/get_release_version.py
       - name: Download release artifacts
-        uses: actions/download-artifact@v3
+        uses: actions/download-artifact@v4
         with:
-          name: rad_cli_release
+          pattern: rad_cli_*
+          merge-multiple: true
```

Comment:
What's causing this to be needed?

Reply:
I explained before. This is because of v4 action change.